### ISSUE TITLE
Hard Coded folder access

### ISSUE THREAD

**Issue by MikeLippincott** [WayScience/pyroptosis_signature_image_profiling#26]
https://github.com/WayScience/pyroptosis_signature/blob/e9537650b0e39a9b9c19df2a47ba3db3eb2f177c/utils/loaddata_utils.py#L102

Could we please make this a dynamic option? I named my folder differently.

**Comment by jenna-tomkinson** [comment]
@MikeLippincott 

I agree this is too hard-coded. I have assigned this to myself so I can make this change in the next PR, thank you for bringing it up.
